doc/classes/Object.xml

```diff
@@ -153,18 +181,73 @@
 				A [code]signal[/code] can only be connected once to a [code]method[/code]. It will throw an error if already connected, unless the signal was connected with [constant CONNECT_REFERENCE_COUNTED]. To avoid this, first, use [method is_connected] to check for existing connections.
 				If the [code]target[/code] is destroyed in the game's lifecycle, the connection will be lost.
 				Examples:
-				[codeblock]
-				connect("pressed", self, "_on_Button_pressed") # BaseButton signal
-				connect("text_entered", self, "_on_LineEdit_text_entered") # LineEdit signal
-				connect("hit", self, "_on_Player_hit", [ weapon_type, damage ]) # User-defined signal
-				[/codeblock]
+				[codeblocks]
+				[gdscript]
+				var button = Button.new()
+				button.pressed.connect(_on_Button_pressed) # is the same as:
+				button.connect("pressed", self, "_on_Button_pressed") # BaseButton signal
```

Comment:
Since you made things explicit by instantiating new nodes (which I'm a bit conflicted about as it makes the examples much more verbose, but I guess it's better to be explicit), you can remove the redundant trailing comments.